From 933c309e1267cb9e8a47d76780a7e947fb2bf6cd Mon Sep 17 00:00:00 2001
From: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date: Fri, 27 Sep 2024 18:57:43 +0200
Subject: [PATCH 82/95] usb: xhci: move Etron PCI IDs to pci_ids.h

The same PCI IDs are needed for the new xhci-pci-etron sub-driver.
Move them to pci_ids.h so they can be re-used instead of having to
copy them over.

Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
---
 drivers/usb/host/xhci-pci.c | 4 ----
 include/linux/pci_ids.h     | 4 ++++
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/usb/host/xhci-pci.c b/drivers/usb/host/xhci-pci.c
index 00fac8b233d2..feb5ff4a9bc1 100644
--- a/drivers/usb/host/xhci-pci.c
+++ b/drivers/usb/host/xhci-pci.c
@@ -34,10 +34,6 @@
 #define PCI_DEVICE_ID_FRESCO_LOGIC_FL1100	0x1100
 #define PCI_DEVICE_ID_FRESCO_LOGIC_FL1400	0x1400
 
-#define PCI_VENDOR_ID_ETRON			0x1b6f
-#define PCI_DEVICE_ID_ETRON_EJ168		0x7023
-#define PCI_DEVICE_ID_ETRON_EJ188		0x7052
-
 #define PCI_DEVICE_ID_VIA_VL805			0x3483
 
 #define PCI_DEVICE_ID_INTEL_LYNXPOINT_XHCI		0x8c31
diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index 92ffc4373f6d..43263f7cdd2a 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -2595,6 +2595,10 @@
 
 #define PCI_VENDOR_ID_REDHAT		0x1b36
 
+#define PCI_VENDOR_ID_ETRON		0x1b6f
+#define PCI_DEVICE_ID_ETRON_EJ168	0x7023
+#define PCI_DEVICE_ID_ETRON_EJ188	0x7052
+
 #define PCI_VENDOR_ID_WCHIC		0x1c00
 #define PCI_DEVICE_ID_WCHIC_CH382_0S1P	0x3050
 #define PCI_DEVICE_ID_WCHIC_CH382_2S1P	0x3250
-- 
2.43.0

